src/thor/triplegbuilder.cc

```diff
@@ -1727,6 +1794,9 @@ void TripLegBuilder::Build(
     // must be done after the edge's shape has been added.
     SetHeadings(trip_edge, controller, directededge, trip_shape, begin_index);
 
+    // Add landmarks in the directededge to the trip leg
+    AddLandmarks(graphreader, trip_edge, controller, directededge, trip_shape, begin_index);
```

Comment:
we should pass `graphtile` here since thats the one that contains `directededge` or we can refactor the shape stuff above to pull out the getting of the edgeinfo and just pass the edgeinfo in, your call